litmus-portal/backend/graphql-server/graph/model/models_gen.go

```diff
@@ -17,6 +17,17 @@ type Cluster struct {
 	ClusterType        string  `json:"cluster_type"`
 }
 
+type WorkflowRun struct {
+	WorkflowRunID string `json:"workflow_run_id" bson:"workflow_run_id"`
+	WorkflowID    string `json:"workflow_id" bson:"workflow_id"`
+	ClusterName   string `json:"cluster_name" bson:"cluster_name"`
+	LastUpdated   string `json:"last_updated" bson:"last_updated"`
+	ProjectID     string `json:"project_id" bson:"project_id"`
+	ClusterID     string `json:"cluster_id" bson:"cluster_id"`
+	WorkflowName  string `json:"workflow_name" bson:"workflow_name"`
+	ExecutionData string `json:"execution_data" bson:"execution_data"`
+}
+
```

Comment:
You can seperate the struct for bson to mongo.go and convert mongo struct to gql struct whenever required.
Ref- clustercreate struct

Reply:
I don't think we are having a similar usecase for the Cluster struct where both the mongo and gql structs are required at the same time. In this case I need both at the same time, hence I have to initalize the same thing twice.

Reply:
Can you point out the places?

Reply:
https://github.com/litmuschaos/litmus/pull/1841#discussion_r467490919

Reply:
We don't have to modify this auto-gen struct. It will remove if we regenerate again.

Reply:
I agree that it will be removed once regenerated but as general coding practice we should not be repeating the same code if we can reuse it, the models dir can be reused for mongo too without any changes. The regerate problem is only during dev but once in prod we won't have to regenerate the schema that frequently